Krytskaya.feature

```diff
@@ -0,0 +1,53 @@
+Feature: 1.1. Contacts
+
+Background:
+    Given user clicked on Notifications "Contacts" 
+
+Scenario Outline: 1.1.1 All external links should be active and right
+    Given user scrolled Contacts to bottom app bars
+    When user clicked on "<Button>" icon
+    Then open a new window in browser 

```

Comment:
In "then" paragraph we use "shoul be" construction (in all test in this file)

Reply:
Then should be open a new window in browser?

Reply:
> 
Yes, it exactly what I mean